docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -283,8 +283,8 @@ Note that in this case the build itself runs in a Docker container too, so you d
 
 [TIP]
 ====
-By default, the native image will be generated using the `quay.io/quarkus/ubi-quarkus-native-image:{graalvm-version}-java8` Docker image which is only compatible with JDK 8.
-If you want to build a native image compatible with JDK 11, you will have to use the `-Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:{graalvm-version}-java11` build argument.
+By default, the native image will be generated using the `quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor}` Docker image which is only compatible with JDK 8.
+If you want to build a native image compatible with JDK 11, you will have to use the `-Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor}` build argument.
```

Comment:
The image URL will be the same for both JDK here, there's no reason to keep that tip in its current state (but maybe it could be rewritten differently).

Reply:
I dropped it. We want to deprecate Java 8 so let's move everyone to JDK 11.

Reply:
Hum. I thought a bit more about it and came up with another version of it.